scripts/k6/stream-api/common.js

```diff
@@ -14,15 +14,16 @@ const PARALLEL_REQS_PER_USER = __ENV.K6_PARALLEL_REQS_PER_USER || 10;
 
 // Constant VUs / iterations count scenario
 const CONST_VIRTUAL_USERS = __ENV.K6_CONST_VIRTUAL_USERS || 1000;
-const ITERATIONS = __ENV.K6_ITERATIONS || 1000000;
-const TIMEOUT = __ENV.K6_TIMEOUT || "20m";
+const CONST_TOTAL_REQUESTS = __ENV.K6_CONST_TOTAL_REQUESTS || 1000000;
+const CONST_TIMEOUT = __ENV.K6_CONST_TIMEOUT || "20m";
 
 // Ramping scenario
-const MAX_VIRTUAL_USERS = __ENV.K6_MAX_VIRTUAL_USERS || 1000;
-const RAMPING_DURATION = __ENV.K6_RAMPING_DURATION || "2m";
-const STABLE_RATE_DURATION = __ENV.K6_STABLE_RATE_DURATION || "2m";
+const RAMPING_MAX_VIRTUAL_USERS = __ENV.K6_RAMPING_MAX_VIRTUAL_USERS || 1000;
+const RAMPING_UP_DURATION = __ENV.K6_RAMPING_UP_DURATION|| "2m";
+const RAMPING_STABLE_DURATION = __ENV.K6_RAMPING_STABLE_DURATION || "10m";
+const RAMPING_DOWN_DURATION = __ENV.K6_RAMPING_DOWN_DURATION || "2m";
 
-const SYNC_MODE = __ENV.STREAM_SYNC_MODE || "disabled"; // disabled / cache / disk
+const SYNC_MODE = __ENV.STREAM_SYNC_MODE || "disk"; // cache / disk
 const SYNC_WAIT = __ENV.STREAM_SYNC_WAIT || "1"; // 1 = enabled, 0 = disabled
```

Comment:
I know, but there is no "disabled" mode.

Reply:
I though that we are not able to `PATCH` the sink without storage API master token. @michaljurecko is it possible just to uncomment that and it would work? I am not sure that the settings are applied